com.unity.renderstreaming/Tests/Runtime/RenderStreamingInternalTest.cs

```diff
@@ -751,6 +751,71 @@ public IEnumerator ResendOfferUntilGotAnswer(TestMode mode)
             target2.Dispose();
         }
 
+        [TestCase(TestMode.PublicMode, ExpectedResult = null)]
+        [TestCase(TestMode.PrivateMode, ExpectedResult = null)]
+        [UnityTest, Timeout(30000), LongRunning]
+        public IEnumerator DeleteFailedPeers(TestMode mode)
+        {
+            MockSignaling.Reset(true);
```

Comment:
Set true to MockSignaling.Reset method if you want to use PrivateMode.

```
public static void Reset(bool enablePrivateMode)
```

```suggestion
            MockSignaling.Reset(mode == TestMode.PrivateMode);
```

Reply:
Oh good catch!

I will get that fixed.